ext/openssl/openssl.c

```diff
@@ -624,6 +630,14 @@ static STACK_OF(X509) * load_all_certs_from_file(char *certfile);
 static X509_REQ * php_openssl_csr_from_zval(zval ** val, int makeresource, long * resourceval TSRMLS_DC);
 static EVP_PKEY * php_openssl_generate_private_key(struct php_x509_request * req TSRMLS_DC);
 
+static void php_openssl_rand_seed() /* {{{ */
+{
+	if (!OPENSSL_G(rng_reseeded)) {
```

Comment:
As I said in the bug report, the `PHP_GINIT_FUNCTION` is missing. In this case it means that you are checking uninitialized variable...